cmd/celestia-appd/cmd/migrate_config.go

```diff
@@ -0,0 +1,214 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/celestiaorg/celestia-app/v6/app"
+	"github.com/celestiaorg/celestia-app/v6/pkg/appconsts"
+	"github.com/cometbft/cometbft/config"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	serverconfig "github.com/cosmos/cosmos-sdk/server/config"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// ConfigUpdater defines a function type for applying version-specific updates
+type ConfigUpdater func(*config.Config, *serverconfig.Config) (*config.Config, *serverconfig.Config)
+
+// updateRegistry maps version strings to their corresponding update functions
+var updateRegistry = map[string]ConfigUpdater{
+	"v6": applyV6Config,
```

Comment:
For signalling we don't prepend the v. It's a slight nit but I would like to be consistent with whether to prepend the v or not

Reply:
makes sense

will change in a flup